lib/taskana-core/src/test/java/acceptance/task/QueryTasksAccTest.java

```diff
@@ -788,6 +788,46 @@ void testQueryForOrderByAttachmentRefValueDesc() {
         .isSortedAccordingTo(CASE_INSENSITIVE_ORDER.reversed());
   }
 
+  @WithAccessId(user = "admin")
+  @Test
+  void testQueryForOrderByAttachmentReceivedAsc() {
+    List<TaskSummary> results =
+        TASK_SERVICE
+            .createTaskQuery()
+            .idIn(
+                "TKI:000000000000000000000000000000000008",
+                "TKI:000000000000000000000000000000000052",
+                "TKI:000000000000000000000000000000000054")
+            .orderByAttachmentReceived(ASCENDING)
+            .list();
+
+    assertThat(results)
+        .hasSize(3)
+        .flatExtracting(TaskSummary::getAttachmentSummaries)
+        .extracting(AttachmentSummary::getReceived)
+        .isSorted();
+  }
+
+  @WithAccessId(user = "admin")
+  @Test
+  void testQueryForOrderByAttachmentReceivedDesc() {
+    List<TaskSummary> results =
+        TASK_SERVICE
+            .createTaskQuery()
+            .idIn(
+                "TKI:000000000000000000000000000000000008",
+                "TKI:000000000000000000000000000000000052",
+                "TKI:000000000000000000000000000000000054")
+            .orderByAttachmentReceived(DESCENDING)
+            .list();
+
+    assertThat(results)
+        .hasSize(3)
+        .flatExtracting(TaskSummary::getAttachmentSummaries)
+        .extracting(AttachmentSummary::getReceived)
+        .isSortedAccordingTo(Comparator.<Instant>naturalOrder().reversed());
```

Comment:
Do you need that `<Instant>`?

Question doesn't really matter since this Test will be converted soon :)

Reply:
yes ... or we will get IDE warnings :D